docs/HOW_TO_UPGRADE.md

```diff
@@ -244,3 +244,55 @@ These are custom modules and this is what you have to keep in mind when updating
   #### 2. Decidim Admin Extended ("decidim-admin-extended"):
 
   This module adds the necessary routes, menus and views to show the Type module inside the admin area
+
+### Overrides
+
+  There has been some overrides that must be taken in care:
+
+  * As described in point 3, in module Decidim Process Extended ("decidim-participatory-processes"), there are following overrides:
```

Comment:
Let's modify point 3 then instead of creating a new section no?

Reply:
You're right @tramuntanal , I delete new section to add overrides in correspondent section 3